docs/ide/visual-studio-ide.md

```diff
@@ -18,7 +18,7 @@ ms.workload:
 
 The Visual Studio *integrated development environment* is a creative launching pad that you can use to edit, debug, and build code, and then publish an app. An integrated development environment (IDE) is a feature-rich program that can be used for many aspects of software development. Over and above the standard editor and debugger that most IDEs provide, Visual Studio includes compilers, code completion tools, graphical designers, and many more features to ease the software development process.
 
-Visual Studio is available for Windows and Mac. [Visual Studio for Mac](/visualstudio/mac/) has many of the same features as Visual Studio 2017, and is optimized for developing cross-platform and mobile apps.
+Visual Studio is available for Windows and Mac. [Visual Studio for Mac](/visualstudio/mac/) has many of the same features as Visual Studio 2017, and is optimized for developing cross-platform and mobile apps. There are three editions types of Visual Studio 2017: **Community, Professional, Enterprise**. The **Community Edition** is free to download for the general public and students. Teams working professionally must buy the **Professional Edition**. Corperate enterprises have to buy the **Enterprise Edition**. Free trials are available for both **Professional** and **Enterprise**.
```

Comment:
- Since the editions only apply to the Windows version, please move that discussion to a new paragraph.
- Please don't boldface the list of editions.
- The last three sentences are guidance that's best left to the VS marketing team. Instead, can you provide a link to https://visualstudio.microsoft.com/vs/compare/, a comparison of editions?